### PR TITLE
Unblock e2e CI tests

### PR DESCRIPTION
**Pull request by davidvossel**
It appears that something in the latest KubeVirt release (either v0.54.0 or v0.55.0) has broken capk's CI. 

To address this, we're pinning the KubeVirt version used to a known version that passes our CI (v0.53.0). We should continue to explicitly pin to specific kubevirt releases to avoid regressions like this slipping in without a PR to capk. In the future, we'll update the kubevirt version in a PR which will force the CI e2e test lane to run before we consider merging

```release-note
NONE
```

### CODE REVIEW

**Comment by davidvossel** [comment]
/hold

i want to see CI pass

**Comment by agradouski** [comment]
/lgtm
thanks for fixing it @davidvossel!

File: e2e/create-cluster_test.go [hunk 1]
```diff
@@ -726,7 +726,7 @@ var _ = Describe("CreateCluster", func() {
 
 	})
 
-	It("creates a simple cluster with persistent VMs", Label("persistentVMs"), func() {
+	FIt("creates a simple cluster with persistent VMs", Label("persistentVMs"), func() {
```
Comment (agradouski) [inline e2e/create-cluster_test.go:729]: typo?

**Comment by davidvossel** [comment]
@agradouski I found another a bit more flakiness that I think is attributed to the size of the ubuntu image we use. I rollled out a new ubuntu image that is nearly half the size that might help a bit. I think kubevirtci environment struggles to consistently work with the 23Gi images we were using.

File: e2e/create-cluster_test.go (hunk 1 above)
Comment (davidvossel) [reply e2e/create-cluster_test.go:729]: Ah, i didn't mean to leave that in. Adding the `F` causes that single test to run. It's a test `focus`. I'll fix that real quick

Comment (agradouski) [reply e2e/create-cluster_test.go]: neat!

**Comment by davidvossel** [comment]
we're still having issues with flaky ci runs. I've transitioned our ci away from using rook-ceph for storage to using "local" storage provider. Now that the ubuntu image is smaller, this might work successfully. We'll see how CI behaves in the next run.

**Comment by davidvossel** [comment]
/hold cancel

CI has passed twice in a row now. The final change I made was to remove use of rook-ceph in favor of the local pvc provisioner. That lowers the CI memory and storage overhead even further, which i believe might be contributing to our flakiness.

We might still encounter future flakiness here if noisy neighbors are impacting us. Hopefully my prow pr [1] will resolve that. For now this PR should at least improve our situation.

1. https://github.com/kubevirt/project-infra/pull/2222

**Comment by davidvossel** [comment]
@agradouski can i get one more lgtm?

**Comment by agradouski** [comment]
thanks @davidvossel - awesome effort digging up the root cause!

**Pull request merged**